test-e2e/build/fixtures/no-deps-_build/package.json

```diff
@@ -9,12 +9,17 @@
       [
         "bash",
         "-c",
-        "echo \"#!/bin/bash\necho $cur__name\" > ./_build/$cur__name"
+        "echo 'print_endline \"#{self.name}\"' > ./_build/test.ml"
```

Comment:
Running this test on Windows, I get: 
```
File "E:/esy2/test-e2e/build/fixtures/no-deps/node_modules/.cache/_esy/store/b/no_deps-1.0.0-d569ef49/test.ml", line 1, characters 14-16:
Error: Unbound value no
```

Looking at the generated `test.ml`, I see this:
```
print_endline no-deps
```

So it looks like we're losing the quotes somehow. I think the `\"` is getting escaped at some point - but its possible this is be a Windows-specific issue (there could be an extra round of escaping when we send it to `esy-bash`). I see this test failing on TravisCI, too, but could be a different issue. If it's windows-specific, we can ignore it for now, and make the fix to `esy-bash`.

Reply:
I have started to add `test.ml` files that we can copy over instead.